deploy/addons/ingress/ingress-rc.yaml

```diff
@@ -77,7 +77,7 @@ spec:
     spec:
       terminationGracePeriodSeconds: 60
       containers:
-      - image: gcr.io/google_containers/nginx-ingress-controller:0.9.0-beta.15
+      - image: quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.9.0-beta.16
```

Comment:
Any reason why this changed to quay.io?

Reply:
@r2d4 In this release, Ingress-nginx team published the image to `quay.io`, but I don't know why this release published to `quay.io`, maybe we can ask member about this release changed.

Reply:
@r2d4 Currently I can't find any reason in the Ingress-nginx project, this PR need to approve?

Reply:
Yeah thats fine.